src/Compilers/CSharp/Test/Symbol/Symbols/PartialPropertiesTests.cs

```diff
@@ -4185,7 +4329,122 @@ partial class C
+            comp.VerifyEmitDiagnostics(
+                // (3,9): warning CS0414: The field 'C.X' is assigned but its value is never used
+                //     int X = 1;
+                Diagnostic(ErrorCode.WRN_UnreferencedFieldAssg, "X").WithArguments("C.X").WithLocation(3, 9),
+                // (4,16): error CS0548: 'C.P': property or indexer must have at least one accessor
+                //     public int P
+                Diagnostic(ErrorCode.ERR_PropertyWithNoAccessors, "P").WithArguments("C.P").WithLocation(4, 16),
+                // (6,9): error CS1014: A get or set accessor expected
+                //         Console.Write(X);
+                Diagnostic(ErrorCode.ERR_GetOrSetExpected, "Console").WithLocation(6, 9),
+                // (6,16): error CS1014: A get or set accessor expected
+                //         Console.Write(X);
+                Diagnostic(ErrorCode.ERR_GetOrSetExpected, ".").WithLocation(6, 16),
+                // (6,17): error CS1014: A get or set accessor expected
+                //         Console.Write(X);
+                Diagnostic(ErrorCode.ERR_GetOrSetExpected, "Write").WithLocation(6, 17),
+                // (6,22): error CS1513: } expected
+                //         Console.Write(X);
+                Diagnostic(ErrorCode.ERR_RbraceExpected, "(").WithLocation(6, 22),
+                // (6,24): error CS8124: Tuple must contain at least two elements.
+                //         Console.Write(X);
+                Diagnostic(ErrorCode.ERR_TupleTooFewElements, ")").WithLocation(6, 24),
+                // (6,25): error CS1519: Invalid token ';' in class, record, struct, or interface member declaration
+                //         Console.Write(X);
+                Diagnostic(ErrorCode.ERR_InvalidMemberDecl, ";").WithArguments(";").WithLocation(6, 25),
+                // (8,1): error CS1022: Type or namespace definition, or end-of-file expected
+                // }
+                Diagnostic(ErrorCode.ERR_EOFExpected, "}").WithLocation(8, 1));
+
+            var tree = comp.SyntaxTrees[0];
+            var model = comp.GetSemanticModel(tree);
+            var node = tree.GetRoot().DescendantNodes().OfType<IdentifierNameSyntax>().Where(name => name.ToString() == "X").Last();
+            Assert.Equal(SyntaxKind.TupleElement, node.Parent!.Kind());
+            var symbolInfo = model.GetSymbolInfo(node);
+            Assert.Null(symbolInfo.Symbol);
+            Assert.Empty(symbolInfo.CandidateSymbols);
+        }
+
+        [Fact]
+        public void LangVersion_01()
+        {
+            var source = """
+                partial class C
+                {
+                    public partial int P { get; set; }
+                    public partial int P { get => 1; set { } }
+
+                    public partial int this[int i] { get; }
+                    public partial int this[int i] { get => i; }
+                }
+                """;
+
+            var comp = CreateCompilation(source);
```

Comment:
Please use RegularNext, so that we capture that the version introducing the feature is C# 13 specifically